src/user/dto/user-info.dto.ts

```diff
@@ -0,0 +1,27 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class UserInfoDTO {
```

Comment:
O nome desse DTO tem que ser assim pra seguir o padrão:

```suggestion
export class GetUserInfoResponse {
```